
<!DOCTYPE html>
<html lang="en">
<head>
  <link href='//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,400italic' rel='stylesheet' type='text/css'>

    <link rel="stylesheet" type="text/css" href="../theme/stylesheet/style.min.css">

  <link rel="stylesheet" type="text/css" href="../theme/pygments/github.min.css">
  <link rel="stylesheet" type="text/css" href="../theme/font-awesome/css/font-awesome.min.css">





  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <meta name="HandheldFriendly" content="True" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <meta name="robots" content="" />


    <meta name="author" content="Nelle" />
    <meta name="description" content="" />
<meta property="og:site_name" content="Nelle"/>
<meta property="og:type" content="blog"/>
<meta property="og:title" content="Nelle"/>
<meta property="og:description" content=""/>
<meta property="og:locale" content="en_US"/>
<meta property="og:url" content=".."/>

  <title>Nelle &ndash; Curriculum Vitae</title>

</head>
<body>
  <aside>
    <div>
      <a href="..">
        <img src="../theme/img/nelle.png" alt="" title="">
      </a>
      <h1><a href=".."></a></h1>


      <nav>
        <ul class="list">
          <li><a href="../pages/about.html#about">About me</a></li>
          <li><a href="../pages/vitae.html#vitae">Curriculum Vitae</a></li>

        </ul>
      </nav>

      <ul class="social">
        <li><a class="sc-github" href="https://github.com/NelleV" target="_blank"><i class="fa fa-github"></i></a></li>
        <li><a class="sc-twitter" href="https://twitter.com/nvaroqua" target="_blank"><i class="fa fa-twitter"></i></a></li>
      </ul>
    </div>


  </aside>
  <main>






<article class="single">
  <header>
    <h1 id="vitae">Curriculum Vitae</h1>
  </header>
  <div>
    <div class="section" id="employment">
<h2>Employment</h2>
<p><strong>Moore-Sloan Data Science Fellow</strong>
Berkeley Institute for Data Science
University of California, Berkeley
<em>August 2016 - present</em></p>
<p><strong>Postdoctoral Scholar</strong>
Department of Statistics
University of California, Berkeley
<em>June 2016 - present</em></p>
</div>
<div class="section" id="publications">
<h2>Publications</h2>
<p>Asterisk indicates dual first-author position. Open access articles are linked
below.</p>
<p><strong>Varoquaux*</strong>, N., Cole, B.*, Gao, C.*, Pierroz, G., Baker, C., Patel, D., Madera, M.,
Jeffers, T., Hollingsworth, J., Sievert, J., Yoshinaga, Y., Owiti, J., Singan, V., De-
Graaf, S., Xu, L., Blow, J. M., Harrison, M., Visel, A., Jansson, C., Niyogi, K. K.,
Hutmacher, R., Coleman-Derr, D., O’Malley, R., Taylor, J., Dahlberg, J., Vogel,
J. P., Lemaux, P. G., and Purdom, E. (Submitted). Lifecycle transcriptomics of
field-droughted Sorghum bicolor reveals rapid biotic and metabolic changes</p>
<p>Geiger, R. S., Fiore-Gartland, B., <strong>Varoquaux, N.</strong>, and Cabasse-Mazel, C.
(Under review). Mapping jupyter's orbits: The local infrastructural
assemblages of a software development ecosystem.</p>
<p>Servant, N.*, <strong>Varoquaux, N.*</strong>, Heard, E., Vert, J.-P., Barillot, E. (2018).
Effective normalization for copy number variation in Hi-C data</p>
<p><strong>Varoquaux, N.</strong>  Unfolding the Genome: The Case Study of <em>P. falciparum</em>.
(2018). <em>International journal of biostatistics</em></p>
<p>Bunnik, E. B.*, Cook, K. B.*, <strong>Varoquaux, N.</strong>, Batugedara, G., Prudhomme,
J., Cort, A., Shi, L., Andolina, C., Ross, L. S., Brady, D., Fidock, D. A.,
Nosten, F., Tewari, R., Sinnis, P.,  Ay, F., Vert, J.-P. Noble, W. S., &amp; Le
Roch, K. G. (2018). Changes in genome organization of parasite-specific gene
families during the Plasmodium transmission stages. <em>Nature Communications</em> 9,
1910</p>
<p>Giordano, R.*, Liu, R.*, <strong>Varoquaux, N.*</strong>, Jordan, M. I., Broderick, T. (2017).
Measuring Cluster Stability for Bayesian Nonparametrics Using the Linear
Bootstrap, <em>NIPS 2017 Advances in Approximate Bayesian Inference Workshop</em></p>
<p>Servant, N., <strong>Varoquaux, N.</strong>, Lajoie, B. R., Viara, E., Chen, C. J., Vert,
J.-P., Heard, E., Dekker, J., Barillot, E. (2015). HiC-Pro: an optimized and flexible
pipeline for Hi-C data processing, <em>Genome Biology</em> 16 (1), 1-11</p>
<p><strong>Varoquaux, N.</strong>, Liachko, I., Ay, F., Burton, J. N., Shendure, J., Dunham,
M. J., Vert, J.-P. &amp; Noble, W. S.  (2015). Accurate identification of
centromere locations in yeast genomes using Hi-C. <em>Nucleic acids research</em>,
43 (11), 5331-5339</p>
<p>Ay, F., Vu, T. H., Zeitz, M. J., <strong>Varoquaux, N.</strong>, Carette, J. E., Vert,
J.-P., Hoffman, A. R. &amp; Noble, W. S. (2015). Identifying multi-locus chromatin
contacts in human cells using tethered multiple 3C. <em>BMC Genomics</em>. 16 (1), 121</p>
<p><strong>Varoquaux, N.</strong>, Ay, F., Noble, W. S., Vert, J.-P. (2014).
A statistical approach for inferring the 3D structure of the genome.
<em>Bioinformatics</em> 30 (12), i26-i33</p>
<p>Ay, F., Bunnik, E., <strong>Varoquaux, N.</strong>,  Vert, J.-P., Noble, W. S., &amp; Le Roch, K.
G. (2014). Multiple dimensions of epigenetic gene regulation in the malaria
parasite <em>Plasmodium falciparum</em>. <em>Bioessays</em> 36 (2015),  37 (2), pp.182-194</p>
<p>Ay, F.*, Bunnik, E. M.*, <strong>Varoquaux, N.*</strong>, Bol, S. B., Prudhomme, J., Vert,
J.-P., Noble, W. S., Le Roch, K. G. (2014) Three-dimensional modeling of the
<em>P. falciparum genome</em> during the erythrocytic cycle reveals a strong
connection between genome architecture and gene expression. <em>Genome research</em>
24 (6), 974-988</p>
<p>Costello, A. C, Heiser, L. M., Georgii, E., Gönen, M., Menden, M. P., Wang, N.
J., Bansal, M., Ammad-ud-din, M., Hintsanen, P., Khan, S. A., Mpindi, J.-P.,
Kallioniemi, O., Honkela, A., Aittokallio, T.,  Wennerberg, K., <strong>NCI DREAM
Community</strong>, Collins, J. J., Gallahan, D., Singer, D., Saez-Rodriguez, J.,
Kaski, S., Gray, J. W., &amp; Stolovitzky, G. (2014). A community effort to assess
and improve drug sensitivity prediction algorithms. <em>Nature biotechnology</em> 32
(12), 1202</p>
<p>Magnin, M., Moreau, G., <strong>Varoquaux, N.</strong>, Vialle, B., Reid, K. Conley, M., &amp;
Gehwolf, S. (2012). MarkUs: An Open-Source Web Application to Annotate Student
Papers On-Line <em>ASME 2012 11th Biennial Conference on Engineering Systems
Design and Analysis</em></p>
<p>Magnin, M., Moreau, G., Tourre, V., <strong>Varoquaux, N.</strong>, Vialle, B. (2012).
Apports de l’application web libre MarkUs dans le processus de correction de
travaux en informatique. <em>Intégration Technologique et Nouvelles Perspectives
d’Usage</em>, 210</p>
</div>

  </div>
</article>

    <footer>
<p>&copy; Nelle </p>
<p>    Powered by <a href="http://getpelican.com" target="_blank">Pelican</a> - <a href="https://github.com/alexandrevicenzi/flex" target="_blank">Flex</a> theme by <a href="http://alexandrevicenzi.com" target="_blank">Alexandre Vicenzi</a>
</p>    </footer>
  </main>





<script type="application/ld+json">
{
  "@context" : "http://schema.org",
  "@type" : "Blog",
  "name": " Nelle ",
  "url" : "..",
  "image": "",
  "description": ""
}
</script>
</body>
</html>